recipes/sdl_mixer/cmake/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
test_v1_package are no longer necessary, it is better to not add them, specially for new recipes (this under ``cmake`` is a new recipe)

Reply:
No need to change it now, but just FYI. Less is more, specially when the backlog in ConanCenter is so big.